docs/interview/interviewer/coding-interview-guide.mdx

```diff
@@ -3,7 +3,171 @@ id: coding-interview
+#### Symbols
+
+- ✅ = Do
+- ❌ = Don't
+- ⚠️ = Situational
+
+# General Tips for a Coding Interview
+## What to do before the interview?
+
+### About the problems
+
+1. ✅ Create a list with various exercises.
+
+In the worst case, your interviewee will have already solved the problem you had prepared, so having a repertoire of problems is ideal to face this situation. You can start with 3 problems and increase them over time. This will also save you time looking for another problem, allowing you to start the interview as quickly as possible. You can create your list using different platforms. We highly recommend using leetcode as a platform for solving and searching for problems. Additionally, if you want to quickly filter the problems you can use, you can access the discord server [cscareers.dev](https://cscareers.dev/discord) and enter the #leetcode-bot channel, where you can get a list of problems filtered by the company by typing !problems <company>.
+
+2. ✅ Look for different solutions to the problems you have gathered.
+
+Remember, there's an infinity of possible solutions for a problem. If you are using leetcode, it is advisable to check the discussion section where you can review solutions from different people. It's highly likely you'll find an algorithm, data structure, etc., completely different from what you had thought or how you solved the problem. Knowing a variety of solutions will allow you to guide your interviewee to the correct solution.
+
+3. ✅ Choose problems from different topics.
+
+If you are using leetcode, you can review the topics related to the problems by simply clicking on the "Related Topics" tab. This way, you can avoid unintentionally repeating topics.
+
+### About you
+
+4. ✅ Prepare a brief introduction.
+
+You don't need to prepare something very complex; you could mention:
+
+- ✅ Your name.
+- ⚠️ If you want to simulate a real interview even more, where you work.
+- ⚠️ The area you work in.
+- ❌ Mention something classified as confidential.
+
+### About the interviewee
+
+5. ✅ Review the CV/resume.
+
+Request the interviewee's CV/resume in advance; this will allow you to know a little about what they have done, what they know, and what they don't. It will also help you choose the most appropriate problem for them to solve.
+
+6. ⚠️ Check if there is feedback from previous interviews.
+
+If the interviewee has had previous interviews, feedback will let you know which points to pay more attention to, aiming to see if the interviewee has improved compared to previous interviews.
+
+### Preparing for the interview
+
+7. ✅ Prepare everything you need.
+
+If you will use a collaborative code editor, an online text document, or some other tool, have all the links and tools you will use handy. If you need an online code editor, we recommend using:
+
+- [codeshare](https://codeshare.io/)
+- [collabedit](http://collabedit.com/)
+
+8. ⚠️ Wait a reasonable amount of time for the interviewee to arrive.
```

Comment:
Can we do more specific here? Maybe just put 20 minutes, we don't want our interviewers to wait the whole hour just waiting.